presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/alluxio/AlluxioProtoUtils.java

```diff
@@ -113,6 +143,96 @@ private static SortingColumn fromProto(alluxio.grpc.table.layout.hive.SortingCol
         return Optional.of(new HiveBucketProperty(property.getBucketedByList(), (int) property.getBucketCount(), sortedBy));
     }
 
+    private static Optional<BigDecimal> fromMetastoreDecimal(@Nullable Decimal decimal)
```

Comment:
Move private functions to the end of a class. Move public ones to the beginning.